src/app/authentication/forgot-password/forgot-password.component.html

```diff
@@ -0,0 +1,27 @@
+<app-header [onlyLogo]="true"></app-header>
+<div id="top-container">
+    <div class="content-container mt-5">
+        <div class="col-md-6 card p-5 m-auto mt-5">
+            <div id="card-content">
+                <h1>FORGOT PASSWORD</h1>
+                <div class="dropdown-divider mt-4"></div>
+
+                <form (ngSubmit)="onSubmitEmailForm()" [formGroup]="forgotPasswordForm">
+                    <label class="w-100">Enter your email address
+                        <input class="form-control mt-3 p-4"
+                               formControlName="email"
+                               placeholder="Email"
+                               type="text">
+                    </label>
+                    <button [disabled]="forgotPasswordForm.invalid" class="btn btn-primary w-100 mt-3 py-3"
+                            type="submit">
+                        <i class="fas fa-paper-plane mr-2"></i>
+                        Submit
+                    </button>
+                </form>
```

Comment:
It is OK, but can be better. We've made <app-action-button> component for this case when you have async action on a button click. 
You can set an observable to call on a button click and then you don't need to worry about setting spinners and all of that. 

You don't need to change it (how you like it), but the steps would be:

1. change `onSubmitEmailForm` method to return an observable (and remove `.subscribe()`)
2. remove spinners from `onSubmitEmailForm` method
2. remove set action button component to call `onSubmitEmailForm.bind(this)` on click`
3. set button template reference variable (e.g. #submitButton) and set form `ngSubmit` action as `(ngSubmit)="submitButton.click()"`.

This way, we have no big overlay spinners, which we should avoid on actions like this, and use it only where the screen needs to be blocked for a reason (e.g. signing a transaction, opening arkane, etc.)